### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -8,9 +8,7 @@ updates:
   - package-ecosystem: "npm"
     directory: "/" # ToDo: check if this updates example-app/
     schedule:
-      interval: "weekly"
-      day: "monday"
-      time: "09:00"
+      interval: "monthly"
```
Comment (bharathkkb) [inline .github/dependabot.yml:11]: Any particular reason for this?

File: dist/index.js [hunk 2]
```diff
@@ -1767,7 +1767,7 @@ function onceStrict (fn) {
 // We use any as a valid input type
 /* eslint-disable @typescript-eslint/no-explicit-any */
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.toCommandValue = void 0;
+exports.toCommandProperties = exports.toCommandValue = void 0;
```
Comment (bharathkkb) [inline dist/index.js:1770]: Do we need to check this in? The build dist PR should create this after we update lockfile.

File: setup-google-cloud-sdk/src/index.ts [hunk 3]
```diff
@@ -164,9 +164,10 @@ export function parseServiceAccountKey(
       "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/service-account-email"
     }
     `;
+    let errorMsg = (error instanceof Error) ? error.message : error as string;
```
Comment (bharathkkb) [inline setup-google-cloud-sdk/src/index.ts:167]: Could we make this a function that takes in an unknown type and returns an Error? That way we can reuse it in other places.

File: .github/dependabot.yml (hunk 1 above)
Comment (averikitsch) [reply .github/dependabot.yml:11]: I want to propose monthly updates since the weekly updates are time consuming.

File: dist/index.js (hunk 2 above)
Comment (averikitsch) [reply dist/index.js:1770]: Oops, will revert

File: setup-google-cloud-sdk/src/index.ts (hunk 3 above)
Comment (averikitsch) [reply setup-google-cloud-sdk/src/index.ts]: done

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**